cmd/eval-dev-quality/cmd/evaluate_test.go

```diff
@@ -576,6 +576,60 @@ func TestEvaluateExecute(t *testing.T) {
 					assert.Contains(t, data, "Run 1/3")
 					assert.Contains(t, data, "Run 2/3")
 					assert.Contains(t, data, "Run 3/3")
+
+					assert.NotRegexp(t, `Run \d+/\d+ for model`, data)
+				},
+				"golang-summed.csv": nil,
+				"models-summed.csv": nil,
+				"README.md":         nil,
+				filepath.Join("symflower_symbolic-execution", "golang", "golang", "plain.log"): func(t *testing.T, filePath, data string) {
+					assert.Equal(t, 3, strings.Count(data, `Evaluating model "symflower/symbolic-execution"`))
+				},
+			},
+		})
+		validate(t, &testCase{
+			Name: "Sequential",
```

Comment:
Test, that this is sequentiel.

Reply:
The log output validation is different between `interleaved` and `sequential` so we are testing already that the execution is truthfully `sequential`